harbor/metadata.csv

```diff
@@ -0,0 +1,4 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+harbor.projects.count,gauge,,object,,The number of Harbor projects.,0,harbor,nb projects
```

Comment:
Would help to provide a description like this one: https://github.com/DataDog/integrations-core/blob/master/kubernetes_state/metadata.csv#L53

Seems by default the metric will be displayed as `avg` value (always 1).